portal-api/api-tests/api-test-users.js

```diff
@@ -45,169 +45,169 @@ const testUsers = [
     firstname: 'first',
     surname: 'last',
     timezone: 'Europe/London',
-    username: 'no-roles',
+    username: 'no-roles@ukexportfinance.gov.uk',
     email: 'no-roles@ukexportfinance.gov.uk',
     password: 'P@ssword1234',
     roles: [],
     bank: {},
   },
   {
-    username: 'HSBC-maker-1',
+    username: 'hsbc-maker-1@ukexportfinance.gov.uk',
```

Comment:
Should `username` _not_ be an email? Not sure if this has changed since I last saw this 😊